tabbycat/venues/views.py

```diff
@@ -78,6 +78,7 @@ def formset_valid(self, formset):
         if self.instances:
             for category in self.instances:
                 category.tournament = self.tournament
+                category.save()
```

Comment:
Just curious, what was the behaviour before this change? Did this omission effectively render the line above it a no-op, or did it do stuff intermittently?

Reply:
No-op if I remember.